src/unix/fs.c

```diff
@@ -120,6 +120,95 @@ static ssize_t uv__fs_fdatasync(uv_fs_t* req) {
 }
 
 
+static ssize_t uv__fs_clamp_max_path_len(ssize_t len) {
```

Comment:
I believe we now have a similar thing in, which will conflict, can you please rebase and adjust?